scripts/prepare_headers.py

```diff
@@ -19,9 +19,13 @@ def prepare(indir, outfile):
         os.path.join(indir, "tss2_tpm2_types.h"), mode="r", encoding="utf-8"
     ).read()
 
+    s += io.open(os.path.join(indir, "tss2_tcti.h"), mode="r", encoding="utf-8").read()
```

Comment:
This is just another way to do the same thing, I find pathlib to be a "less lines" approach and have been enjoying it, so wanted to share.

```python
    s += pathlib.Path(indir, "tss2_tcti.h").read_text()
```

Reply:
I ended up re-working the regex prepare_headers.py file to more organized and made this change when doing it. As well as added FAPI and cleaned up the testing code a bit.